### PR TITLE
Add Notification badge on Filter screen

### PR DESCRIPTION
**Pull request by saketPandhare-Webonise**

### CODE REVIEW

File: SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift [hunk 1]
```diff
@@ -195,4 +196,11 @@ extension MoveVC: FilterScreenDelegate {
         ///Just reload the existing table view
         tableViewMove.reloadData()
     }
+
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift:199]: Please add the method documentation.

File: SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift [hunk 2]
```diff
@@ -145,6 +145,7 @@ class MoveVC: NavigationController {
         let filterVC = storyBoard.instantiateViewController(withIdentifier: ControllerIdentifier.FILTER_VC) as! FilterVC
         filterVC.filterScreenDelegate = self
         filterVC.hidesBottomBarWhenPushed = true
+
```
Comment (vineet-webonise) [inline SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift:148]: Please remove extra space if not required.

File: SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift (hunk 1 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift:198]: Done

File: SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift (hunk 2 above)
Comment (saketPandhare-Webonise) [reply SUNFLOWR/SUNFLOWR/MoveVC/MoveVC.swift]: Done

**Pull request merged**